src/WebJobs.Script.Scaling/AppServiceSettings.cs

```diff
@@ -259,19 +260,31 @@ public static byte[] RuntimeEncryptionKey
             {
                 if (_runtimeEncryptionKey == null)
                 {
-                    var value = Environment.GetEnvironmentVariable("WEBSITE_ENCRYPTION_KEY");
+                    var value = Environment.GetEnvironmentVariable("WEBSITE_AUTH_ENCRYPTION_KEY");
                     if (string.IsNullOrEmpty(value))
                     {
-                        throw new InvalidOperationException("MIssing WEBSITE_ENCRYPTION_KEY environment variable");
+                        throw new InvalidOperationException("MIssing WEBSITE_AUTH_ENCRYPTION_KEY environment variable");
```

Comment:
small nit: this was already here, but the message has an uppercase I in `MIssing`

Reply:
good catch - fixed.